src/main.ts

```diff
@@ -30,7 +29,6 @@ async function run() {
       ...context.repo,
       deployment_id: parseInt(deploymentId),
       state,
-      log_url: logUrl,
       target_url: url,
```

Comment:
I guess we may as well rename `target_url` to `log_url`, right?

Reply:
Hi @tony.  I think the fundamental problem is that I want a URL for a deployed app, which should live in `environment_url`, but that field isn't included in `deployment_status` events in Github (at least not in a way that is visible inside Github workflows). `log_url` should replace `target_url` as in @ypresto's PR, but doing so means the data in `target_url` isn't passed to the `deployment_status` event. So honestly I don't think my PR should go in, the real fix is probably to find out why `environment_url` isn't being passed around, and  fix it or propose a change, and in the meantime it's necessary to hack what should be the environment URL into the free-form `description` field.

Reply:
@jimCresswell Thanks for the response.  I don't fully understand your use case, likely due to my own naiveite of Deployment API at the moment.

So if you do develop a one-size-fits-all case, that would be good (if you have time). If not no problem.   We'll also see if @ypresto has anything to say